apps/wallet/src/ui/app/experimentation/feature-gating.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { GrowthBook } from '@growthbook/growthbook';
+
+const DEFAULT_DEV_API_KEY = 'key_dev_dc2872e15e0c5f95';
+
+export default class FeatureGating {
+    #growthBook: GrowthBook;
+
+    constructor() {
+        // Create a GrowthBook context
+        this.#growthBook = new GrowthBook();
+    }
+
+    public async init() {
+        const apiKey = process.env.GROWTH_BOOK_API_KEY ?? DEFAULT_DEV_API_KEY;
```

Comment:
Why even make this configurable? Do we imagine having separate keys?

Reply:
Right now we have separate keys for production and development. Making this configurable in case we have separate environments for TestNet and DevNet

Reply:
Can we just encode both of the keys in the source and flip them accordingly?